Resources/Prototypes/GameRules/roundstart.yml

```diff
@@ -254,12 +254,73 @@
   parent: BaseGameRule
   components:
   - type: BasicStationEventScheduler
+    scheduledGameRules:
+    - id: ClosetSkeleton
+    - id: MouseMigration
+    - id: SlimesSpawn
+    - id: BluespaceArtifact
+    - id: ClericalError
+    - id: IonStorm
+    - id: KingRatMigration
+    - id: CockroachMigration
+    - id: SpiderClownSpawn
+    - id: VentClog
+    - id: GasLeak
+    - id: ImmovableRodSpawn
+    - id: BreakerFlip
+    - id: PowerGridCheck
+    - id: BluespaceLocker
+    - id: NinjaSpawn
+    - id: BureaucraticError
+    - id: DragonSpawn
+    - id: SpiderSpawn
+    - id: MimicVendorRule
+    - id: LoneOpsSpawn
+    - id: RevenantSpawn
+#    - id: MassHallucination    # What the fuck is this? Is it its own whoosiwhatsit and not an event?
+    - id: SleeperAgents
+    - id: AnomalySpawn
+    - id: SolarFlare
+    - id: KudzuGrowth
+    - id: ZombieOutbreak
+    - id: RandomCargoGiftsRule
+    - id: RandomUnknownShuttleRule
 
 - type: entity
   id: RampingStationEventScheduler
   parent: BaseGameRule
   components:
   - type: RampingStationEventScheduler
+    scheduledGameRules:
+    - id: ClosetSkeleton
+    - id: MouseMigration
+    - id: SlimesSpawn
+    - id: BluespaceArtifact
+    - id: ClericalError
+    - id: IonStorm
+    - id: KingRatMigration
+    - id: CockroachMigration
+    - id: SpiderClownSpawn
+    - id: VentClog
+    - id: GasLeak
+    - id: ImmovableRodSpawn
+    - id: BreakerFlip
+    - id: PowerGridCheck
+    - id: BluespaceLocker
+    - id: NinjaSpawn
+    - id: BureaucraticError
+    - id: DragonSpawn
+    - id: SpiderSpawn
+    - id: MimicVendorRule
+    - id: LoneOpsSpawn
+    - id: RevenantSpawn
+    - id: SleeperAgents
+    - id: AnomalySpawn
+    - id: SolarFlare
+    - id: KudzuGrowth
+    - id: ZombieOutbreak
+#    - id: CargoGiftMasterRule
```

Comment:
same here?

Reply:
This one was just an accident, but I thought it was funny that survival doesn't get the mercy gifts.